documentation/docs/30-advanced/20-hooks.md

```diff
@@ -92,9 +92,9 @@ Note that `resolve(...)` will never throw an error, it will always return a `Pro
 
 ### handleFetch
 
-This function allows you to modify (or replace) a `fetch` request that happens inside a `load` function that runs on the server (or during pre-rendering).
+This function allows you to modify (or replace) a `fetch` request that happens inside a `load` or `actions` function that runs on the server (or during pre-rendering).
```

Comment:
Even though the export is `actions`, it reads a bit weirdly as a plural so I removed the `s`

```suggestion
This function allows you to modify (or replace) a `fetch` request that happens inside a `load` or `action` function that runs on the server (or during pre-rendering).
```